modules/console/pages/config/security/authentication.adoc

```diff
@@ -1,61 +1,367 @@
 = Authentication
-:description: Redpanda Console supports authentication using OAuth 2.0 or OIDC for external identity providers.
-:page-aliases: console:single-sign-on/authentication.adoc, manage:security/console/authentication.adoc
-:page-categories: Management, Security, Redpanda Console
+:description: Redpanda Console supports login authentication using OIDC (OAuth 2.0) and basic authentication—empowered by user impersonation for secure, unified API access.
+:page-aliases: console:single-sign-on/authentication.adoc, manage:security/console/authentication.adoc, console:config/security/plain.adoc, console:config/security/okta.adoc, console:config/security/generic-oidc.adoc, console:config/security/keycloak.adoc, console:config/security/github.adoc, console:config/security/google.adoc, console:config/security/azure-ad.adoc
+
+// ========================AUTOMATED TESTS===================================
+// The comments in this file are used to run automated tests of the documented steps. Tests are run using GitHub Actions on each pull request that changes this file in the upstream repository. For more details about the testing tool we use, see https://doc-detective.com/.
+
+// (test start {"id": "console-authentication", "description": "Generate screenshot of Console login screen"})
+// (step {"action":"runShell", "command": "docker compose up -d --wait", "workingDirectory": "../console-login", "timeout": 50000, "exitCodes": [0,1]})
+// (step {"action":"wait", "duration": 20000})
+// (step {"action":"goTo", "url": "http://localhost:8080/login"})
+// (step {"action":"wait", "duration": 10000})
+// (step {"action":"saveScreenshot", "path": "login.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"runShell", "command": "docker compose down -v", "workingDirectory": "../console-login"})
+// (test end)
 
 [NOTE]
 ====
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-Redpanda Console supports authentication using OAuth 2.0 or OIDC for external identity providers, such as:
+This topic describes how to enable authentication in Redpanda Console and explains how it interacts with the Redpanda APIs. Authentication allows users to log in to Redpanda Console and provides an option to reuse those login credentials to authenticate to the connected Redpanda cluster, ensuring that API requests are executed under the user's identity.
+
+image::console:login.png[]
+
+Redpanda Console authenticates to the Redpanda APIs in one of two ways:
+
+- *User-impersonation*: When you log in using OIDC (OpenID Connect) or basic authentication, a JSON Web Token (JWT) is generated. This token is used for API calls to the Kafka API, Admin API, and Schema Registry, ensuring that actions are performed under your identity and are properly audited in Redpanda.
```

Comment:
The JWT is really an implementation detail, I think this may confuse users a bit. Because the token itself is not used for authenticating between Console and Redpanda but is rather the session between Console Backend and Frontend. This session token contains the user's credentials for authenticating between Console <-> Redpanda and that could be either SASL with username+password or OIDC with the id token (that we will also refresh if expired)